BD-3560: Condense beacon and geofence support info (#8237)
braze-inc/braze-docs: _docs/_partners/message_personalization/location/gimbal.md

> [Gimbal](https://gimbal.com/) enables you to perfect your marketing relevance using location data. Their location SDK paired with geofencing software and beacons power relevant, personalized, proximity-aware mobile experiences.

Combine your beacon or geofence support with Braze's targeting and messaging features to learn more about your user's physical actions and message them accordingly. This partnership integration opens up an array of use cases for:

- **Marketing:** Send contextually relevant messaging and build experiential consumer journeys.
- **Competitive Analysis:** Set up triggers around competitive locations to understand consumer trends and patterns.
- **Audience Insights:** Understand your users' visitation behaviors and further segment based on those learnings.

{% alert note %}

> This will work the same for Gimbal’s beacons as well as their geofence solutions.

Reviewer: What is "this" here? Also, we should avoid making possessive of other company names when we can.

```suggestion
This integration works the same for Gimbal beacons and Gimbal geofence solutions.
```
